greeter/README.md

```diff
@@ -418,23 +423,23 @@ this.configure('greeters.ChineseGreeter').to({nameFirst: false});
 
 ### Step 5: Try it Out!
 
-Restart your application, try endpoint `GET /greeting/{name}`, with `Accept-Language` set to `zh`.
+Restart your application, try endpoint `GET /greeting/{name}`, with `language` set to `zh`.
 
-You will see message `'name, hello'` printed instead of `'hello, name'`.
+You will see message `'hello, name'` printed instead of `'name, hello'`.
 
 Related documentation:
 [Configuration by convention](https://loopback.io/doc/en/lb4/Context.html#configuration-by-convention)
 
 ## Part 3: Enable Caching
 
-To reduce the cost of calculation, we'd like to use a caching server for our greeting application. We will show how to bind a caching server, interceptor and observer for your application as an example of contributing microservice using LB4.
+To reduce the cost of calculation, we'd like to use a caching service for our greeting application. We will show how to bind a caching service, interceptor and observer for your application as an example of contributing a microservice using LB4.
 
-The caching server will cache the greeting messages for a certain amount of time. And cached message will be swept out when they are expired.
+The caching service will cache the greeting messages for a certain amount of time. And cached messages will be swept out when they are expired.
 
-Again, we are using `bind` and `injection` to bind our app with Interceptor and Observer:
+Again, we are using `bind` and `injection` to bind our app with an Interceptor and Observer:
 
-- Interceptors: are reusable functions to provide aspect-oriented logic around method invocations. (should make it more intuitive)
-- Observer: is a LoopBack application that has its own life cycles at runtime. Observer allows artifacts to participate in the life cycles. The transition of states is triggered by two functions: start() and stop().
+- Interceptors: are reusable functions that provide [aspect-oriented](https://en.wikipedia.org/wiki/Aspect-oriented_programming) logic around method invocations.
+- Observer: is a LoopBack application that has its own life cycles at runtime. An Observer allows artifacts to participate in the life cycles. The transition of states is triggered by two functions: `start()` and `stop()`.
```

Comment:
> Observer: is a LoopBack application that has its own life cycles at runtime. An Observer allows artifacts to participate in the life cycles. The transition of states is triggered by two functions: `start()` and `stop()`.

Observer: A LoopBack application that has its own life cycles at runtime. An [Observer interface](https://loopback.io/doc/en/lb4/Life-cycle.html#the-lifecycleobserver-interface) allows artifacts to participate in the life cycles. The transition of states is triggered by two functions: `start()` and `stop()`.